exasol_integration_test_docker_environment/testing/api_test_environment.py

```diff
@@ -58,27 +58,28 @@ def _get_default_test_environment(self, name: str, ports: Ports):
             ports=ports,
         )
 
-    def spawn_docker_test_environment_with_test_container(self, name: str,
-                                                          test_container_content: TestContainerContentDescription,
-                                                          additional_parameter: Dict[str, Any] = None) \
-            -> ExaslctDockerTestEnvironment:
+    def spawn_docker_test_environment_with_test_container(
+            self,
+            name: str,
+            test_container_content: TestContainerContentDescription,
+            additional_parameter: Dict[str, Any] = None,
+    ) -> ExaslctDockerTestEnvironment:
         if additional_parameter is None:
             additional_parameter = dict()
         ports = Ports.random_free()
-        on_host_parameter = self._get_default_test_environment(name, ports)
-        docker_db_image_version = on_host_parameter.docker_db_image_version
-        ports = on_host_parameter.ports
-        on_host_parameter.environment_info, on_host_parameter.clean_up = \
+        on_host = self._get_default_test_environment(name, ports)
```

Comment:
hm, on_host is not really specific. In case of naming, I prefer precision before shortness.

Reply:
Will be reverted in next push.